### PR TITLE
BsSP-TG-9: Employees create and edit

### PR DESCRIPTION
**Pull request by Martinzancocchia**

### CODE REVIEW

File: src/Components/Employees/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    if (search.match('id=')) {
+      const id = search.substring(search.indexOf('id=') + 3);
+
+      fetch(`${process.env.REACT_APP_API_URL}/employees/${id}`, {
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: body
+      })
+        .then((response) => response.json())
+        .then((response) => console.log(response));
```
Comment (LucianaPrat) [inline src/Components/Employees/Form/index.js:33]: Please delete console.log

File: src/Components/Employees/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    } else {
+      fetch(`${process.env.REACT_APP_API_URL}/employees`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: body
+      })
+        .then((response) => response.json())
+        .then((response) => console.log(response));
```
Comment (LucianaPrat) [inline src/Components/Employees/Form/index.js:41]: Please delete console.log

File: src/Components/Employees/index.js [hunk 3]
```diff
@@ -1,27 +1,87 @@
...
       });
   }, []);
 
+  // const updateEmployee = () => {
+  //   fetch(`${process.env.REACT_APP_API_URL}/employees`)
+  //     .then((response) => response.json())
+  //     .then((response) => {
+  //       saveEmployees(response.data);
+  //     });
+  // };
```
Comment (LucianaPrat) [inline src/Components/Employees/index.js:24]: there should be no comments

**Review by LucianaPrat — DISMISSED** [review]

**Comment by NicolasGulam** [comment]
Did some comments in trello

**Review by NicolasGulam — APPROVED** [review]

**Review by LucianaPrat — APPROVED** [review]

**Review by nailaghigo — APPROVED** [review]
nice work! :)

**Pull request merged**